### ISSUE TITLE
Updated ReadMe - Thoughts?

### ISSUE THREAD

**Issue by KieranRobson** [techno-tim/littlelink-server#147]
https://github.com/forks-by-kieran/littlelink-server/blob/master/README.md

I cleaned up the readme, thought Id get some thoughts before opening a PR.

**Comment by timothystewart6** [comment]
Thank you! I like the direction you are going for!  The only feedback is that I would still like to include some of the documentation for some of the variables that aren't BUTTONS (metrics, logs, etc).  Otherwise people will ask how to implement it.

**Comment by KieranRobson** [comment]
@timothystewart6 Added them under a configuration heading, feel free to give some feedback!

**Comment by KieranRobson** [comment]
Would you like the other emojis added such as the whale and graph for the smaller headers or just leave the current emojis in the larger ones?

I'll open a pr once I make sure all links are good and there aren't any typos!

**Comment by timothystewart6** [comment]
Thanks!  `h1` and h2` but not `h3`  Sorry to be so picky!

**Comment by Choubakawa** [comment]
Looks great! 👍 

But are we authorized to use the logo of Littlelink? Should we make a new one (based on the original) to differentiate the 2 projects?

On the same idea, I've started to list all the variables in a wiki, it's not finished yet but I don't know if it's useful as it's only more read-friendly. 
I share the link here:
https://github.com/Choubakawa/littlelink-server/wiki/Available-variables
